examples/Example21/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+# SPDX-FileCopyrightText: 2022 CERN
+# SPDX-License-Identifier: Apache-2.0
+
+if(NOT TARGET G4HepEm::g4HepEm)
+  message(STATUS "Disabling example21 (needs G4HepEm)")
+  return()
+endif()
+
+if(Geant4_FOUND)
+  if(NOT Geant4_gdml_FOUND)
+    message(STATUS "Disabling example21 (needs Geant4 with GDML support)")
+    return()
+  endif()
+  if(NOT Geant4_flush_FOUND)
+    message(STATUS "Disabling example21 (needs G4VFastSimulationModel::Flush interface available starting with geant4-11-00-ref-01)")
+    return()
+  endif()
```

Comment:
This example doesn't use `G4VFastSimulationModel::Flush` but `G4VTrackingManager`, which is available since Geant4 11.0

Reply:
fixed